doc/release/README.chplenv

```diff
@@ -238,14 +238,17 @@ Optional Settings
    options are:
 
    CHPL_TASKS:
-        qthreads       : run tasks using Sandia's Qthreads package
-        fifo           : run tasks to completion, in FIFO order
-        massivethreads : run tasks using U Tokyo's MassiveThreads package
+        qthreads       : use Sandia's Qthreads package
+        fifo           : use POSIX threads
+        massivethreads : use U Tokyo's MassiveThreads package
+        muxed          : use Cray-specific lightweight threading
```

Comment:
We've been trying to move away from the term "proprietary", but I could add a clause saying that this is for the module only